kotlin-audio/src/main/java/com/doublesymmetry/kotlinaudio/notification/NotificationManager.kt

```diff
@@ -316,9 +440,19 @@ class NotificationManager internal constructor(
     }
 
     fun invalidate() {
-        internalNotificationManager?.invalidate()
-        mediaSessionConnector.invalidateMediaSessionQueue()
-        mediaSessionConnector.invalidateMediaSessionMetadata()
+        if (invalidateThrottleCount++ == 0) {
+            scope.launch {
+                internalNotificationManager?.invalidate()
```

Comment:
still feels like we're spamming the invalidate methods, but I guess if they don't complain and it generally works its fine.